specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -17,15 +17,6 @@
       "apim_key": []
     }
   ],
-  "x-ms-parameterized-host": {
```

Comment:
Please revert the changes of other APIs.

Reply:
done.